scripts/seeding/seeding_prep.py

```diff
@@ -307,6 +320,47 @@ def _retrive_token_id(file_matching_pattern_str, current_tick):
     return token_id
 
 
+def _calc_steth_out_to_peg_tick(pool, wbtc, ebtc, collateral):
+    path = [wbtc, ebtc]
+    step_increase = 0.05
+    min_wbtc_amount = 0.5
+    max_wbtc_amount = 4.5  # @note swapping anything above 4.5 certainly will have negative price impact
```

Comment:
One idea to make this reusable in the following weeks is to have this value based on the balances of the pool. Otherwise we will need to manipulate the amount manually in future instances. On the other hand, chances are that this is the last time that this is possible,